Source/iOS/OIDAuthorizationUICoordinatorIOS.m

```diff
@@ -45,6 +46,7 @@ @implementation OIDAuthorizationUICoordinatorIOS {
   BOOL _authorizationFlowInProgress;
   __weak id<OIDAuthorizationFlowSession> _session;
   __weak SFSafariViewController *_safariVC;
+   SFAuthenticationSession *_authenticationVC;
```

Comment:
Thanks! I added some #ifdefs to support both iOS 11 and 10 as the base SDK. PTAL.